util/app-config/src/tests/graceful_shutdown.bats

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bats
+bats_load_library 'bats-assert'
+bats_load_library 'bats-support'
+
```

Comment:
can we add a test case, some sub-services terminated before receiving exit notification, and then the `wait_all_ckb_services_exit` also works for this scenario, maybe also double confirm the log messages.

Reply:
Yes, we can.